pkg/controller/nodeipam/node_ipam_controller.go

```diff
@@ -96,13 +97,13 @@ func NewNodeIpamController(
 	// Cloud CIDR allocator does not rely on clusterCIDR or nodeCIDRMaskSize for allocation.
```

Comment:
also change the Fatalf on line 93 above?

Reply:
Updated.